deploy/kubernetes/snapshot/validation-webhook/README.md

```diff
@@ -0,0 +1,62 @@
+1. Clone [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter) repo. The following steps should be run from the same external-snapshotter folder.
+2. Run the `create-cert.sh` script. More details about the script are mentioned below.
+```
+./deploy/kubernetes/webhook-example/create-cert.sh --service snapshot-validation-service --secret snapshot-validation-secret --namespace default
+```
+- The script takes 3 arguments - `service` which is name of the k8s service for the webhook, `secret` which is the name of the k8s secret which will hold RSA key and certificate, `namespace` which is the namespace where the pods, secret and service related to this webhook is deployed. It these arguments are not provided, the default values are `service - admission-webhook-example-svc`, `secret - admission-webhook-example-certs`, `namespace - default`.
+- First the script creates a pem encoded private RSA key and certificate signing request using openssl, the same are named server-key.pem and server.csr files respectively.
+- Next a [certificate signing request](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/) is created using server.csr content as input with kubernetes.io/kubelet-serving signer. To understand more about signers, please refer [this](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#kubernetes-signers). 
+- `kubectl approve` command is run to approve the CSR. This automatically generates a signed certificate in the Certificate Signed Request under `.status.certificate` field.
+- A kubernetes secret is created with the secret name provided while runnning the script in the given namespace. The secret holds two keys in its data, `cert.pem` which holds the X.509 certificate and `key.pem` which holds the private RSA key. This secret will be used by snapshot validation webhook once deployed.
+3. Fetch the certificate authority of the cluster where you are deploying the webhook. You will either find it or tha path leading to it in your KUBECONFIG file, under the name `certificate-authority` under your cluster. Fetch the content of the file (it begins and ends with `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----` respectively). Encode the file `base64 -i <file>.pem`.
+4. Add the base64 encoded value in `deploy/kubernetes/webhook-example/admission-configuration.yaml` file in front of `caBundle:` paramater.
+5. Change the namespace to where you prefer to deploy the webhook in `admission-configuration.yaml, rbac-snapshot-webhook.yaml, webhook.yaml` files under `deploy/kubernetes/webhook-example`/` directory
```

Comment:
nit, "files under `deploy/kubernetes/webhook-example/`" instead of "files under `deploy/kubernetes/webhook-example`/`"